### PR TITLE
Related to T936: update the show tech-support scripts

### PR DESCRIPTION
**Pull request by shafiev**
Added network related mode output
Added system - os related (cpu, memory, file system space, pkgs etc.) related mode output

### CODE REVIEW

File: functions/tech-support [hunk 1]
```diff
@@ -68,241 +70,256 @@ fi
 export PATH=/sbin:/usr/sbin:$PATH
 
 header Show Tech-Support  
+  
 header CONFIGURATION 
+  
+function show_version { 
+  if cli-shell-api validateTmplPath entitlement; then
```
Comment (dmbaturin) [inline functions/tech-support]: This is one of the cases where cleanup was needed badly. The "entitlement" commands haven't existed in years.

File: functions/tech-support [hunk 2]
```diff
@@ -68,241 +70,256 @@ fi
...
+    header Entitlement
+    show entitlement
+  fi
 
-if cli-shell-api validateTmplPath entitlement; then
```
Comment (dmbaturin) [inline functions/tech-support:73]: Mixing formatting changes with logic changes is a very bad style. How is the reviewer supposed to follow the logic changes when the file is littered with inconsequential formatting changes?

File: functions/tech-support [hunk 3]
```diff
@@ -314,257 +331,299 @@ if cli-shell-api existsActive protocols bgp; then
...
-  show wan-load-balance 
-  header Wan Load Balance Status 
-  show wan-load-balance status
-  header Wan Load Balance Connection
-  show wan-load-balance connection
-else
-  echo "Wan Load Balance is not configured"
-fi
+  header VRRP
+  # XXX: not checking if configured, we'd have to walk all VIFs
```
Comment (dmbaturin) [inline functions/tech-support]: A long solved problem: VRRP is in "high-availability vrrp".

File: functions/tech-support [hunk 4]
```diff
@@ -314,257 +331,299 @@ if cli-shell-api existsActive protocols bgp; then
...
 
+function show_dns { 
+  header DNS 
+  if cli-shell-api existsActive service dns; then
+    header DNS Dynamic Status
+    show dns dynamic status
+    header DNS Forwarding Statistics
+    show dns forwarding statistics
+    header DNS Forwarding Nameservers  
+    show dns forwarding nameservers
```
Comment (dmbaturin) [inline functions/tech-support]: This command doesn't exist anymore.

File: functions/tech-support [hunk 5]
```diff
@@ -314,257 +331,299 @@ if cli-shell-api existsActive protocols bgp; then
...
-    header /etc/ipsec.secrets
-    cat /etc/ipsec.secrets
+  header IPSec
+  if cli-shell-api existsActive vpn ipsec; then
+    header IPSec Status
+    show vpn ipsec status
+    header IPSec sa
+    show vpn ipsec sa
+    header IPSec sa Detail
+    show vpn ipsec sa detail
```
Comment (dmbaturin) [inline functions/tech-support]: This command no longer exists.

File: functions/tech-support [hunk 6]
```diff
@@ -314,257 +331,299 @@ if cli-shell-api existsActive protocols bgp; then
...
+    header IPSec sa Statistics
+    show vpn ipsec sa statistics
```
Comment (dmbaturin) [inline functions/tech-support]: This command no longer exists, which would be trivial to find out by running it.

File: functions/tech-support (hunk 1 above)
Comment (shafiev) [reply functions/tech-support]: I have deleted this block

File: functions/tech-support (hunk 2 above)
Comment (shafiev) [reply functions/tech-support:73]: Delete this block

File: functions/tech-support (hunk 5 above)
Comment (shafiev) [reply functions/tech-support]: Delete this command

File: functions/tech-support (hunk 6 above)
Comment (shafiev) [reply functions/tech-support]: Delete this command

File: functions/tech-support (hunk 4 above)
Comment (shafiev) [reply functions/tech-support]: Deleted this command

File: functions/tech-support (hunk 3 above)
Comment (shafiev) [reply functions/tech-support]: Maybe replace is with show vrrp detail command?

Comment (dmbaturin) [reply functions/tech-support]: Oh right, removing a comment that describes a problem solves the problem.
You can see that before gathering outputs from every component, the script tries to find out if it's configured to begin with. Like in `if cli-shell-api existsActive protocols ospf;`. 

VRRP doesn't fit that pattern. The comment vaguely explains why: it was hard to do. I clearly say why it's no longer the case: VRRP is a single top level path "high-availability vrrp".

File: functions/tech-support (hunk 5 above)
Comment (dmbaturin) [reply functions/tech-support]: So you've kept all "run show vpn ipsec sa *" commands that no longer exist and removed the one that does? https://github.com/vyos/vyatta-op/blob/9d6c8e3426145818802fc62e75168925bc4323cf/functions/tech-support#L416-L422

Or I'm misunderstanding your intent?

File: functions/tech-support (hunk 3 above)
Comment (shafiev) [reply functions/tech-support]: I have added checking for VRRP  .

File: functions/tech-support (hunk 5 above)
Comment (shafiev) [reply functions/tech-support]: Please check .There was a just some misunderstanding, sorry

**Pull request closed without merging**